### DISCUSSION TITLE
ActiveDirectory Integration?

### DISCUSSION THREAD

**Discussion by T-Justin96** [polonel/trudesk#503] [Ideas]
Hey, i love the System!

I would suggest adding an (LDAP) Active Directory integration.

The features are endless...

- Sync users between all your services
- Sync permissions, passwords, mail-addresses, "Teams" and much more between all your services!

I would really love this!

**Comment by polonel** [comment]
The community edition had an LDAP import feature that was removed in `v1.2.0` as this feature is listed as being part of the cloud version.
